guides/android_sdk_guia_de_instalacao.md

```diff
@@ -22,7 +22,7 @@ Se você tiver alguma dúvida ou sugestão, não hesite em me contatar através
 ## Ferramentas de linha de comando
 
 - Abra a [página de download do android
-  sdk](https://developer.android.com/studio#downloads)
+  sdk](https://developer.android.com/studio#command-line-tools-only)
```

Comment:
You are -- of course -- correct! I assumed since the old link was still `#downloads` that the `id` attributes wouldn't be translated, and I didn't test. Thanks for the correction! Fixed in [9614c4b](https://github.com/mesaquen/android-cmdline-tools-guide/pull/1/commits/9614c4b54bfb6cfe029be32fa47489f9bd6e65b8)!